app/javascript/turbo_modal/scrim_controller.js

```diff
@@ -15,15 +15,27 @@ class ScrimController extends Controller {
 
   show(event) {
     delete this.scrimTarget.dataset.hidden;
+    document.body.style.height = "100vh";
+    document.body.style.overflow = "hidden";
   }
 
   hide(event) {
     this.scrimTarget.dataset.hidden = "";
+    document.body.style.height = "unset";
+    document.body.style.overflow = "unset";
   }
 
   onClick(event) {
     window.dispatchEvent(new Event("scrim:hide"));
   }
+
+  disconnect() {
+    this.scrimTarget.dataset.hidden = "";
+    document.body.style.height = "unset";
+    document.body.style.overflow = "unset";
```

Comment:
Duplication here could be extracted to a private method?

Reply:
Good idea, have updated that